app.json

```diff
@@ -1,106 +1,129 @@
+      "required": true
+    },
+    "EDXORG_CLIENT_ID": {
+      "description": "The OAuth client ID configured in the edX instance.",
+      "required": true
+    },
+    "EDXORG_CLIENT_SECRET": {
+      "description": "The OAuth client secret configured in the edX instance.",
+      "required": true
+    },
+    "ELASTICSEARCH_HTTP_AUTH": {
+      "descriptin": "Basic auth settings for connecting to Elasticsearch"
+    },
+    "ELASTICSEARCH_INDEX": {
+      "description": "Index to use on Elasticsearch"
+    },
+    "ELASTICSEARCH_URL": {
+      "description": "URL for connecting to Elasticsearch cluster"
+    },
+    "MAILGUN_KEY": {
+      "description": "The token for authenticating against the Mailgun API"
+    },
+    "MICROMASTERS_ADMIN_EMAIL": {
+      "description": "E-mail to send 500 reports to.",
+      "required": false
+    },
+    "MICROMASTERS_EMAIL_HOST": {
+      "description": "Outgoing e-mail settings",
+      "required": false
+    },
+    "MICROMASTERS_EMAIL_PASSWORD": {
+      "description": "Outgoing e-mail settings",
+      "required": false
+    },
+    "MICROMASTERS_EMAIL_PORT": {
+      "description": "Outgoing e-mail settings",
+      "required": false,
+      "value": "587"
+    },
+    "MICROMASTERS_EMAIL_TLS": {
+      "description": "Outgoing e-mail settings",
+      "required": false,
+      "value": "True"
+    },
+    "MICROMASTERS_EMAIL_USER": {
+      "description": "Outgoing e-mail settings",
+      "required": false
+    },
+    "MICROMASTERS_ENVIRONMENT": {
+      "description": "The execution environment that the app is in (e.g. dev, staging, prod)"
+    },
+    "MICROMASTERS_FROM_EMAIL": {
+      "value": "MITx MicroMasters <micromasters-support@mit.edu>"
+    },
+    "MICROMASTERS_SECURE_SSL_REDIRECT": {
+      "description": "Application-level SSL redirect setting.",
+      "required": false,
+      "value": "True"
+    },
+    "MICROMASTERS_USE_S3": {
+      "description": "Use S3 for storage backend (required on Heroku)",
+      "value": "True"
+    },
+    "NEW_RELIC_APP_NAME": {
+      "default": "MicroMasters",
+      "description": "Application identifier in New Relic."
+    },
+    "OPEN_EXCHANGE_RATES_APP_ID": {
+      "description": "The app ID for the open exchange rates API"
+    },
+    "OPEN_EXCHANGE_RATES_URL": {
+      "value": "https://openexchangerates.org/api/"
+    },
+    "SECRET_KEY": {
+      "description": "Django secret key.",
+      "generator": "secret"
+    },
+    "SENTRY_DSN": {
+      "description": "The connection settings for Sentry"
+    },
+    "STATUS_TOKEN": {
+      "description": "Token to access the status API.",
+      "required": true
     }
+  },
+  "keywords": [
+    "Django",
+    "Python",
+    "MIT",
+    "Office of Digital Learning"
+  ],
+  "name": "micromasters",
+  "repository": "https://github.com/mitodl/micromasters",
+  "scripts": {
+    "postdeploy": "./manage.py migrate"
```

Comment:
It should theoretically only need to be run right after a new deploy, otherwise the app's signals should take care of keeping Elasticsearch up to date. But without a generated `ELASTICSEARCH_INDEX` we probably can't do this step automatically without colliding with other indexes